packages/app/src/components/promo/PromoAction.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div
+    data-cy="promo-action"
+    class="flex flex-row items-center gap-[12px] h-[32px] bg-gray-50 border border-gray-100 rounded w-fit text-sm"
```

Comment:
Forcing a fix height here is forcing the button to overflow due to the button's height plus border.  I am playing around with some options, but not there yet.

Reply:
I had noticed that too and tried to hack it with the `py-[5px]` class on `Button` to offset the button's padding to account for the border pixels